monitor_job.py

```diff
@@ -87,13 +86,11 @@ def record_restream_stats():
 
 
 if __name__ == '__main__':
-    while True:
-        try:
-            record_all_metrics()
-            record_num_inputs()
-            record_restream_stats()
-        except Exception as e:
-            print e
-
-        print "Monitor process sleeping for {}m".format(MINUTES_SLEEP)
-        time.sleep(SECONDS_SLEEP)
+    try:
+        record_all_metrics()
+        record_num_inputs()
+        record_restream_stats()
+    except Exception as e:
+        print e
+
+    print "Monitor process sleeping for {}m".format(MINUTES_SLEEP)
```

Comment:
Maybe update this copy?

Reply:
I think we can get rid of the sleep constants too

Reply:
We are still using it in `record_metric` but I moved the environment variable lookup into that method.  

I then also noticed the we currently have the SLEEP_INTERVAL_MINUTES set to 1 (not the default 10 minutes).  I don't know the history  - maybe 10 minutes is ok.

Reply:
@thebrettd @joemanley201 Do you know why we have SLEEP_INTERVAL_MINUTES set to one and whether it's important to keep it at 1? If we use this Scheduler add-on, the smallest available interval is 10 minutes.

Reply:
No, I don't know :(